spor.c

```diff
@@ -647,10 +647,12 @@ inline static void executeInstr(Instr instr) {
       if(WS_POP()) return;
       // intentional fallthrough
     case RET:
+    {
```

Comment:
These should not be necessary. Can you try without them and let me know the error that happens when they are removed?

Reply:
Oh! Weird that my compiler didn't throw this error. It's complaining about the variable declaration.

Replace
```
U4 cont = Stk_pop(&env.cs);
```

With
```
r = Stk_pop(&env.cs);
```

Then replace the uses of `cont` with `r`.

Reply:
I didn't realize this could be fixed by using `{ ... }`. That's good to know!

Reply:
It works!